stf/tasks/create_components_forecast.py

```diff
@@ -61,7 +61,7 @@ def callback(pj, successful):
 
                 update_status_change(status_id, status_code)
 
-        model_type = ["xgb", "xgb_quantile"]
```

Comment:
xgb shoud remain xgb but lgb should be added to the list

Reply:
I'll change it, thanks!